modules/instrument_builder/jsx/react.questions.js

```diff
@@ -502,6 +502,10 @@ class ListElements extends Component {
     let textSize = 'small';
     // Set the options for the desired type
     switch (newId) {
+      case 'line':
+        newState.Options = {};
+        newState.Name = '';
+        newState.Description = '';
```

Comment:
There was a "blank line" option, why does this now need a linst spec change to work?

Reply:
@driusan when debugging the bug I encountered that the previous state of Options, Name, Description were getting carried over when creating a 'break line' and setting them to default values would allow the 'break line' to work.

Reply:
So does this change anything in the saved file? Also, does this case need a `break`?

Reply:
@driusan forgot to test that. I'll check and report back.

Reply:
@driusan seems to not being adding to .linst file. I'll debug some more and fix. As of now just the frontend error is fixed for rendering the break lines.

Reply:
What about the `break` statement part? Right now this is falling through to the textarea/textbox cases (though I doubt that's the cause of it not saving properly..)

Reply:
@driusan nice catch on the missing break. That part of the code makes the Blank Line render on the frontend of the instrument builder. Otherwise before the code change, it wasn't rendering the Blank Line to the frontend. The new code I just added fixes the save to a .linst file.

Reply:
nice catches.  Do we also need a change to an existing linst spec file, @driusan ?  if so, let's tag update or tag as `needs documentation`

Reply:
Can you confirm @maltheism @coreyAbout  - 
Does this reset (wipe) the last-used Options array?  
It should keep the last options for the next (e.g. dropdown) question added, if at all possible. 
The intended behaviour is that when the user adds `dropdown1:options{a, b, c}` then `blank line` then `dropdown2` -- the options will already be loaded with `{a, b, c}`